recipes/aio-executor/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "aio-executor" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aio-executor-{{ version }}.tar.gz
+  sha256: e9b78a1355cd7beb55bdc367c94fcfe5e8bc188cc457e8e2ab1f4bf2666caf98
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+
+test:
+  imports:
+    - aio_executor
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://github.com/miguelgrinberg/aio-executor/
+  summary: A concurrent.futures.Executor implementation that runs asynchronous tasks in an asyncio event loop.
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
```

Comment:
@jakirkham Let me know if you'd like to be a maintainer as well since you requested it.